docs/public/configuration/README.md

```diff
@@ -1,12 +1,14 @@
 # Configuration
 
-This section contains information about:
+*Click on [SUMMARY](../SUMMARY.md) to view a linked tree structur of all documents
+under "public".*
```

Comment:
This will create a broken link when building the HTML documentation. The SUMMARY.md file won't have any HTML counterpart. I suggest removing this paragraph (and all similar paragraphs in the other documents).

Reply:
Is the name SUMMARY.md magic? If I create a symlink TREE.md --> SUMMARY.md will then TREE.md get an HTML counterpart?

I really want to use the SUMMARY.md since it is very helpful for the users instead walking through the README.md files.

Reply:
No there is no magic, as [the documentation says](https://rust-lang.github.io/mdBook/format/summary.html): "_The summary file is used by mdBook to know what chapters to include, in what order they should appear, what their hierarchy is and where the source files are. Without this file, there is no book._"
In order to have an HTML file, you need to list the Markdown file into SUMMARY.md. And currently, I don't feel at ease with listing SUMMARY.md into itself.

Reply:
I understand what you mean, but the SUMMARY file is really useful when navigating in the document tree. I think it would be wrong to hide it for the readers in Github.

Reply:
The new `docs/public/README.md` is not included in `SUMMARY.md` and I guess that it does not have to be included, and then I can keep the link in that README, but remove it from the other READMEs.

Reply:
Now that I better understand how it works I can see that there were some documents under `specification/test-zones` that were missing.

Reply:
Well it could be an idea to include the `docs/public/README.md` to the SUMMARY.md file. In this way, the landing page would be this README file.

```diff
diff --git a/docs/public/SUMMARY.md b/docs/public/SUMMARY.md
index 33738dda..86fa3090 100644
--- a/docs/public/SUMMARY.md
+++ b/docs/public/SUMMARY.md
@@ -2,6 +2,7 @@
  https://rust-lang.github.io/mdBook/format/summary.html -->
 # Documents in public document tree
 
+- [Introduction](README.md)
 - [Installation](installation/README.md)
     - [Prerequisites](installation/prerequisites.md)
     - [Zonemaster-LDNS](installation/zonemaster-ldns.md)
```

Reply:
If `docs/public/README.md` is included in `SUMMARY.md` then it cannot link to `SUMMARY.md`. I prefer to have it this way. I really want to be able to link to `SUMMARY.md` for Github users.